layout.tt

```diff
@@ -59,6 +61,7 @@
             <ul class="dropdown-menu">
               <li><a href="[%root%]">NixOS</a></li>
               <li><a href="[%root%]nix">Nix</a></li>
+              <li><a href="[%root%]nix-pills">Nix Pills</a></li>
```

Comment:
It seems strange to put this in the drop down menu as a top-level project. Shouldn't it just be linked from the docs page?